bears/general/InvalidLinkBear.py

```diff
@@ -57,6 +59,8 @@ def find_links_in_file(file, timeout, link_ignore_regex):
     def run(self, filename, file,
             timeout: int=DEFAULT_TIMEOUT,
```

Comment:
I see, I forgot to update the `DEFAULT_TIMEOUT` to `DEFAULT_TIMEOUT['anything could go here and it will still return 2, you see thats the beauty of defaultdict']` Thanks for introducing it to me, makman